scripts/deployment/arbitrum-local-template.json

```diff
@@ -9,25 +9,33 @@
             "name": "Arbitrum",
             "rpc": "http://localhost:8547",
             "chain_id": 412346,
-            "finality_period": 3600,
             "l1_messenger": ["ArbitrumL1Messenger", ${l1_messenger_args}],
             "l2_messenger": "ArbitrumL2Messenger",
             "request_manager_arguments": {
                 "claim_stake": 0.00047,
                 "claim_request_extension": 3600,
                 "claim_period": 3600,
-                "challenge_period_extension": 3600
+                "challenge_period_extension": 3600,
+                "lp_margin": 30
             },
             "tokens":
                 [
                     {
                         "token_address": "mintable_token",
+                        "price": 1000,
```

Comment:
even though it's an test config, I think it looks a bit misleading that we choose 1000 here but in the tests we use ..e18 representation. 
I know it doesn't really matter here but we should somehow state clearly what price means in this context.